### PR TITLE
Kubecost docs

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-65915

## Description of changes being made
Initial add of kubecost docs. @russurquhart1 please feel free to make edits/add more context directly, I wasn't sure how much kubecost context to give since it is a separate product with its own docs. There may need to be docs added on kommander UI kubecost integration, if so we can loop in the UI team to help write them.

TODO: we will also need to add licensing information once the kubecost/kommander licensing strategy is finalized.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
@gracedo can we add a description to the release notes for this new feature?

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: layout.pug
+navigationTitle: Centralized Cost Monitoring
+title: Centralized Cost Monitoring
+menuWeight: 7
+beta: true
+excerpt: Monitoring costs of all managed clusters with Kubecost
+---
+
+[Kubecost][https://docs.kubecost.com/] running on Kommander provides centralized cost monitoring for all managed clusters.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:10]:
```suggestion
[Kubecost][https://docs.kubecost.com/], running on Kommander, provides centralized cost monitoring for all managed clusters.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+This feature is provided by default in every Kommander cluster and provides a centralized view of Kubernetes resource spend on all managed clusters. By default, up to 15 days of cost metrics are retained, with no backup to an external store.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:11]:
```suggestion
This feature, installed by default in every Kommander cluster, provides a centralized view of Kubernetes resources used on all managed clusters. By default, up to 15 days of cost metrics are retained, with no backup to an external store.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Centralized Costs
+
+The Kommander cluster collects cost metrics remotely from each managed cluster using Thanos. An overview of cost by workspace and project is shown in the Kommander UI on the respective pages. Further cost analysis and details can be found in the Kubecost UI running on Kommander, that can be accessed at:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:15]:
```suggestion
The Kommander cluster collects cost metrics remotely, using Thanos, from each managed cluster. An overview of cost, by workspace and project, is shown in the respective Kommander UI pages. Further cost analysis and details are found in the Kubecost UI running on Kommander, at:
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+https://<CLUSTER_URL>/ops/portal/kommander/kubecost/detail.html#&agg=cluster
+```
+
+<p class="message--note"><strong>NOTE: </strong>It takes at least 3 hours for each managed cluster to generate sufficient cost data to be aggregated and presented in the UI.</p>
+
+Managed clusters are distinguished by a monitoring ID, that is used to identify clusters in Kubecost.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:23]:
```suggestion
To identify clusters in Kubecost, managed clusters are distinguished by a monitoring ID.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+The monitoring ID corresponds to the kube-system namespace UID of the cluster.
+To find a cluster's monitoring ID, you can go to the Clusters tab on the Kommander UI (in the relevant workspace):
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:25]:
```suggestion
To find a cluster's monitoring ID, go to the **Clusters** tab on the Kommander UI, in the relevant workspace:
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+https://<CLUSTER_URL>/ops/portal/kommander/ui/#/clusters
+```
+
+Select `View Details` on the managed cluster card, and find the monitoring ID under `Monitoring ID (clusterId)`.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:31]:
```suggestion
Select **View Details** on the managed cluster card, and find the monitoring ID under **Monitoring ID (clusterId)**.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+You can also search or filter by monitoring IDs on the Clusters page.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:33]:
```suggestion
You can also search or filter by monitoring IDs on the **Clusters** page.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+Alternatively, you can run a kubectl command, **using the correct cluster's context or kubeconfig**, to look up the cluster's kube-system namespace UID to determine which cluster the costs correspond to:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:35]:
```suggestion
Alternatively, you can run the following  kubectl command, **using the correct cluster's context or kubeconfig**, to look up the cluster's kube-system namespace UID and determine the cluster with the corresponding costs.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+```bash
+$ kubectl get namespace kube-system -o jsonpath='{.metadata.uid}'
+```
+
+### Grafana dashboards
+
+There is also a set of Grafana dashboards with cost metrics that are installed to the centralized Grafana UI:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:43]:
```suggestion
A set of Grafana dashboards with cost metrics are installed to the centralized Grafana UI:
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
+---
+layout: layout.pug
+navigationTitle: Centralized Cost Monitoring
+title: Centralized Cost Monitoring
+menuWeight: 7
+beta: true
+excerpt: Monitoring costs of all managed clusters with Kubecost
+---
+
+[Kubecost][https://docs.kubecost.com/], running on Kommander, provides centralized cost monitoring for all managed clusters.
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:10]: FYI - I'll add this change for linking @gracedo @cmastr 
```
[Kubecost](https://docs.kubecost.com/), running on Kommander, provides centralized cost monitoring for all managed clusters.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
+---
+layout: layout.pug
+navigationTitle: Centralized Cost Monitoring
+title: Centralized Cost Monitoring
+menuWeight: 7
+beta: true
+excerpt: Monitoring costs of all managed clusters with Kubecost
+---
+
+[Kubecost](https://docs.kubecost.com/), running on Kommander, provides centralized cost monitoring for all managed clusters.
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:10]: To open this in a new tab fyi - @cmastr @russurquhart1 
```suggestion
<a href="https://docs.kubecost.com/" target="_blank">Kubecost</a>, running on Kommander, provides centralized cost monitoring for all managed clusters.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+https://<CLUSTER_URL>/ops/portal/kommander/kubecost/detail.html#&agg=cluster
+```
+
+<p class="message--note"><strong>NOTE: </strong>It takes at least 3 hours for each managed cluster to generate sufficient cost data to be aggregated and presented in the UI.</p>
+
+To identify clusters in Kubecost, managed clusters are distinguished by a monitoring ID.
+The monitoring ID corresponds to the kube-system namespace UID of the cluster.
+To find a cluster's monitoring ID, go to the **Clusters** tab on the Kommander UI, in the relevant workspace:
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:25]: does there need to be a comma after `tab on the Kommander UI, `
Instead being just 
go to the **Clusters** tab on the Kommander UI in the relevant workspace
?

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+### Grafana dashboards
+
+A set of Grafana dashboards with cost metrics are installed to the centralized Grafana UI:
+
+```
+https://<CLUSTER_URL>/ops/portal/kommander/monitoring/grafana
+```
+
+These dashboards provide a global view of accumulated costs from all managed clusters.
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:49]: @gracedo 
After looking at grafana, we could probably just add a note about how the user will need to select the 
Kubecost cluster metrics or the Cluster cost & utilization metrics, and the like dashboards on grafana, as cost is not automatically set up on the grafana dashboard upon going to that link.

```suggestion
These dashboards provide a global view of accumulated costs from all managed clusters. From the nav in the above Grafana dashboard, you can select the Kubecost cluster metrics, or the Cluster cost & utilization metrics to view these dashboards.
```

Comment (gracedo) [reply pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md]: Used this sentence instead:
```
From the nav in Grafana, you can find these dashboards by selecting those tagged with `cost`, `metrics`, and `utilization`.
```

File: pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md (hunk 12 above)
Comment (gracedo) [reply pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:25]: @russurquhart1

Comment (cmastr) [reply pages/ksphere/kommander/1.1.0-beta/centralized-cost-monitoring/index.md:25]: no comma.

**Pull request merged**